spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/SecretsConfigProperties.java

```diff
@@ -92,12 +90,9 @@ public String getConfigurationTarget() {
 	 */
 	public List<SecretsConfigProperties.NormalizedSource> determineSources() {
 		if (this.sources.isEmpty()) {
-			return new ArrayList<SecretsConfigProperties.NormalizedSource>() {
-				{
-					add(new SecretsConfigProperties.NormalizedSource(SecretsConfigProperties.this.name,
+			return Collections
```

Comment:
double-brace initialization is a common anti-pattern, replace with a `singletonList`